portal/tasks.py

```diff
@@ -138,6 +141,31 @@ def test_args(*args, **kwargs):
     return "{}|{}".format(",".join(args), json.dumps(kwargs))
 
 
+@celery.task(name="tasks.consume_list", ignore_result=True, soft_time_limit=4)
+def consume_list(id_list, priority):
+    try:
+        time.sleep(random.randint(1, 5))
+        logger.info("priority: {} consuming {}".format(priority, id_list))
+    except SoftTimeLimitExceeded:
+        logger.info("timed out")
+
+
+@celery.task(name="tasks.produce_list")
+def produce_list():
+    j = 0
+    step = 5
+    numlists = 50
+    for i in range(step, (step*numlists)+1, step):
```

Comment:
I think it might be more straightforward to create one range (1-250), then slice from it in the loop

Reply:
it's just PoC, but if you want to submit a patch, that'd be fine :)  as is, you can toggle each independently